api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -0,0 +1,121 @@
+                <not>
+                    <tableExists tableName="patient_program_attribute" />
+                </not>
+            </preConditions>
+             <comment>Creating patient_program_attribute table</comment>
+            <createTable tableName="patient_program_attribute">
+                <column name="patient_program_attribute_id" type="int" autoIncrement="true">
+                    <constraints primaryKey="true" nullable="false" />
+                </column>
+                <column name="patient_program_id" type="int"><constraints nullable="false" /></column>
+                <column name="attribute_type_id" type="int"><constraints nullable="false" /></column>
+                <column name="value_reference" type="text"><constraints nullable="false" /></column>
+                <column name="uuid" type="char(38)"><constraints nullable="false" unique="true" /></column>
+                <column name="creator" type="int"><constraints nullable="false"/></column>
+                <column name="date_created" type="datetime"><constraints nullable="false"/></column>
+                <column name="changed_by" type="int"/>
+                <column name="date_changed" type="datetime" />
+                <column name="voided" type="boolean" defaultValueBoolean="false">
+                    <constraints nullable="false" />
+                </column>
+                <column name="voided_by" type="int" />
+                <column name="date_voided" type="datetime" />
+                <column name="void_reason" type="varchar(255)" defaultValue="null"/>
+            </createTable>
+            <addForeignKeyConstraint constraintName="patient_program_attribute_programid_fk"
+                                     baseTableName="patient_program_attribute" baseColumnNames="patient_program_id"
+                                     referencedTableName="patient_program" referencedColumnNames="patient_program_id" />
+            <addForeignKeyConstraint constraintName="patient_program_attribute_attributetype_fk"
+                                     baseTableName="patient_program_attribute" baseColumnNames="attribute_type_id"
+                                     referencedTableName="program_attribute_type" referencedColumnNames="program_attribute_type_id" />
+            <addForeignKeyConstraint constraintName="patient_program_attribute_creator_fk"
+                                     baseTableName="patient_program_attribute" baseColumnNames="creator"
+                                     referencedTableName="users" referencedColumnNames="user_id" />
+            <addForeignKeyConstraint constraintName="patient_program_attribute_changed_by_fk"
+                                     baseTableName="patient_program_attribute" baseColumnNames="changed_by"
+                                     referencedTableName="users" referencedColumnNames="user_id" />
+            <addForeignKeyConstraint constraintName="patient_program_attribute_retired_by_fk"
```

Comment:
Since the field is voided_by, the constraint name would also be "patient_program_attribute_voided_by_fk" instead of "patient_program_attribute_retired_by_fk"